tests/test_model_loading.py

```diff
@@ -106,6 +106,77 @@ def test_model_loading_from_badpath_raises_FileNotFoundError():
         capellambse.MelodyModel(badpath)
 
 
+def test_split_protocol_returns_path_objects_unchanged():
+    expected = pathlib.Path.cwd()
+
+    handler, actual = capellambse.filehandler.split_protocol(expected)
+
+    assert handler == "file"
+    assert actual == expected
+
+
+@pytest.mark.parametrize(
+    ["url", "expected"],
+    (
+        [
+            ("/path/to/file", pathlib.Path("/path/to/file")),
+            ("file:///path/to/file", pathlib.Path("/path/to/file")),
+            ("file://localhost/path/to/file", pathlib.Path("/path/to/file")),
+        ]
+        if not sys.platform.startswith("win")
+        else [
+            ("C:/path/to/file", pathlib.Path("C:/path/to/file")),
+            (r"C:\path\to\file", pathlib.Path(r"C:\path\to\file")),
+            ("file:///C:/path/to/file", pathlib.Path("C:/path/to/file")),
+            (
+                "file://localhost/C:/path/to/file",
+                pathlib.Path("C:/path/to/file"),
+            ),
+        ]
+    ),
+)
+def test_split_protocol_converts_file_urls_to_paths(url, expected):
```

Comment:
Missing type-annotations:
```suggestion
def test_split_protocol_converts_file_urls_to_paths(url: str, expected: pathlib.Path):
```